docs/usage/faq.md

```diff
@@ -196,3 +196,35 @@ document.body.removeEventListener('click', proxy_f)
 proxy_f.destroy()
 ```
 This also avoids memory leaks.
+
+## How can I use fetch with optional arguments from Python?
+The most obvious translation of the Javascript code won't work:
+```py
+resp = await js.fetch('/someurl', {
+    "method": "POST"
+  , "body": '{ "some" : "json" }'
+  , "credentials": "same-origin"
+  , "headers": { "Content-Type": "application/json" }
+})
+```
+this leaks the dictionary and the `fetch` api ignores the options that we
+attempted to provide. There are two correct ways to do this:
+```py
+from pyodide import to_js
+resp = await js.fetch('example.com/some_api',
+  method= "POST",
+  body= '{ "some" : "json" }',
```

Comment:
does this dictionary also need to be passed to `to_js`?

Reply:
No, because it's a string. I will update the example though because every time I look at it I think the same thing and have to notice the quote marks. Thanks!

Reply:
ah! yeah, it totally got me :)